### ISSUE TITLE
Add to play store

### ISSUE THREAD

**Issue by Donkey-Doug** [jkuester/unlauncher#75]
Perhaps it makes sense to wait until the app matures, but please add the app to the app store as well.

**Comment by jkuester** [comment]
I am going to label this issue as `wontfix`, but leave the issue open. At this point I have no desire to contribute to the Play Store (and be forced to jump through all of Google's hoops to just be able to publish the app). I would rather focus my efforts on free/open methods of distribution (e.g. F-Droid).

That being said, I am open to any serious discussion that folks would like to have about why it might be important to publish Unlauncher in the Play Store. So I will leave this issue open as a place to have that discussion.

**Comment by elSolus** [comment]
Nah, I think it is fine without PleySt0re. Let's not give them a gem.
